trl/trainer/utils.py

```diff
@@ -354,13 +399,26 @@ def __call__(self, features: List[Dict[str, Any]]) -> Dict[str, Any]:
                         padding_value = self.label_pad_token_id
                     elif k.endswith("_attention_mask"):
                         padding_value = 0
+                    elif k.endswith("_pixel_values"):
+                        padding_value = 0  # TODO: check if this is correct
```

Comment:
Anyone have a counter example (ie pixel padding value not equal to 0?)

Reply:
should we add it to the model or method config?

Reply:
idk